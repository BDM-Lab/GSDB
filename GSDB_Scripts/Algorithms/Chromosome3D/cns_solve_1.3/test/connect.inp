remarks test file: connect.inp
remarks  connectivity feature
remarks

topology
  remarks topology
  remarks
  !
  mass TA 1.0
  mass TB 1.0
  mass TC 1.0
  !
  residue GRAP
    atom A type TA end
    atom B type TB end
    atom C type TC end
    atom D type TA end
    atom E type TB end
    atom F type TC end
    atom G type TA end
    atom H type TB end
    atom I type TC end
    atom J type TA end
    atom K type TB end
    atom L type TC end
    !
    bond A B 
    bond B C 
    bond C D
    bond D E
    bond E A
    bond C A
    bond D A
    !
    bond F G 
    bond F G
    bond F G
    bond G H
    ! 
    bond I J
    bond J K
    bond K I
    !
  end
  !
end !topology
!
segment
  name=main
  molecule 
     name=GRAP number=2
  end
end
!
connectivity
    print=true selection=(all)
end

display  $mset

connectivity
    print=true selection=(all)
    setidto=rmsd  
end

show elem ( rmsd ) ( all ) 

connectivity
    print=true selection=(all)
    setidto=rmsd  nsetto=fbeta
end

show elem ( rmsd ) ( all ) 
show elem ( fbeta ) ( all ) 

connectivity
    print=true selection=(all)
    setidto=store1  nsetto=store2 
end

show elem ( store1 ) ( all ) 
show elem ( store2 ) ( all ) 


!
stop
