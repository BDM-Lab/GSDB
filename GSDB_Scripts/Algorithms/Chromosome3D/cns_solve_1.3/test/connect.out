          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:07:50 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>remarks test file: connect.inp 
 CNSsolve>remarks  connectivity feature 
 CNSsolve>remarks 
 CNSsolve> 
 CNSsolve>topology 
 RTFRDR>  remarks topology 
 RTFRDR>  remarks 
 RTFRDR>  ! 
 RTFRDR>  mass TA 1.0 
 RTFRDR>  mass TB 1.0 
 RTFRDR>  mass TC 1.0 
 RTFRDR>  ! 
 RTFRDR>  residue GRAP 
 RESIDUE>    atom A type TA end 
 RESIDUE>    atom B type TB end 
 RESIDUE>    atom C type TC end 
 RESIDUE>    atom D type TA end 
 RESIDUE>    atom E type TB end 
 RESIDUE>    atom F type TC end 
 RESIDUE>    atom G type TA end 
 RESIDUE>    atom H type TB end 
 RESIDUE>    atom I type TC end 
 RESIDUE>    atom J type TA end 
 RESIDUE>    atom K type TB end 
 RESIDUE>    atom L type TC end 
 RESIDUE>    ! 
 RESIDUE>    bond A B 
 RESIDUE>    bond B C 
 RESIDUE>    bond C D 
 RESIDUE>    bond D E 
 RESIDUE>    bond E A 
 RESIDUE>    bond C A 
 RESIDUE>    bond D A 
 RESIDUE>    ! 
 RESIDUE>    bond F G 
 RESIDUE>    bond F G 
 RESIDUE>    bond F G 
 RESIDUE>    bond G H 
 RESIDUE>    ! 
 RESIDUE>    bond I J 
 RESIDUE>    bond J K 
 RESIDUE>    bond K I 
 RESIDUE>    ! 
 RESIDUE>  end 
 RTFRDR>  ! 
 RTFRDR>end !topology 
 CNSsolve>! 
 CNSsolve>segment 
 SEGMENT>  name=main 
 SEGMENT>  molecule 
 MOLECULE>     name=GRAP number=2 
 MOLECULE>  end 
 SEGMENT>end 
 Status of internal molecular topology database:
 -> NATOM=         24(MAXA=     1000000)  NBOND=         28(MAXB=     1000000)
 -> NTHETA=         0(MAXT=     2000000)  NGRP=           2(MAXGRP=   1000000)
 -> NPHI=           0(MAXP=     2000000)  NIMPHI=         0(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve>! 
 CNSsolve>connectivity 
 CONNECT>    print=true selection=(all) 
 SELRPN:     24 atoms have been selected out of     24
 CONNECT>end 
 CONNECt: selected atoms form      8 covalently disconnected set(s)

 CONNECt: list of all atoms in set      1
              (MAIN 1    GRAP A   )
              (MAIN 1    GRAP B   )
              (MAIN 1    GRAP C   )
              (MAIN 1    GRAP D   )
              (MAIN 1    GRAP E   )

 CONNECt: list of all atoms in set      2
              (MAIN 1    GRAP F   )
              (MAIN 1    GRAP G   )
              (MAIN 1    GRAP H   )

 CONNECt: list of all atoms in set      3
              (MAIN 1    GRAP I   )
              (MAIN 1    GRAP J   )
              (MAIN 1    GRAP K   )

 CONNECt: list of all atoms in set      4
              (MAIN 1    GRAP L   )

 CONNECt: list of all atoms in set      5
              (MAIN 2    GRAP A   )
              (MAIN 2    GRAP B   )
              (MAIN 2    GRAP C   )
              (MAIN 2    GRAP D   )
              (MAIN 2    GRAP E   )

 CONNECt: list of all atoms in set      6
              (MAIN 2    GRAP F   )
              (MAIN 2    GRAP G   )
              (MAIN 2    GRAP H   )

 CONNECt: list of all atoms in set      7
              (MAIN 2    GRAP I   )
              (MAIN 2    GRAP J   )
              (MAIN 2    GRAP K   )

 CONNECt: list of all atoms in set      8
              (MAIN 2    GRAP L   )
 CNSsolve> 
 CNSsolve>display  $mset 
 8
 CNSsolve> 
 CNSsolve>connectivity 
 CONNECT>    print=true selection=(all) 
 SELRPN:     24 atoms have been selected out of     24
 CONNECT>    setidto=rmsd 
 CONNECT>end 
 CONNECt: selected atoms form      8 covalently disconnected set(s)

 CONNECt: list of all atoms in set      1
              (MAIN 1    GRAP A   )
              (MAIN 1    GRAP B   )
              (MAIN 1    GRAP C   )
              (MAIN 1    GRAP D   )
              (MAIN 1    GRAP E   )

 CONNECt: list of all atoms in set      2
              (MAIN 1    GRAP F   )
              (MAIN 1    GRAP G   )
              (MAIN 1    GRAP H   )

 CONNECt: list of all atoms in set      3
              (MAIN 1    GRAP I   )
              (MAIN 1    GRAP J   )
              (MAIN 1    GRAP K   )

 CONNECt: list of all atoms in set      4
              (MAIN 1    GRAP L   )

 CONNECt: list of all atoms in set      5
              (MAIN 2    GRAP A   )
              (MAIN 2    GRAP B   )
              (MAIN 2    GRAP C   )
              (MAIN 2    GRAP D   )
              (MAIN 2    GRAP E   )

 CONNECt: list of all atoms in set      6
              (MAIN 2    GRAP F   )
              (MAIN 2    GRAP G   )
              (MAIN 2    GRAP H   )

 CONNECt: list of all atoms in set      7
              (MAIN 2    GRAP I   )
              (MAIN 2    GRAP J   )
              (MAIN 2    GRAP K   )

 CONNECt: list of all atoms in set      8
              (MAIN 2    GRAP L   )
 CNSsolve> 
 CNSsolve>show elem ( rmsd ) ( all ) 
 SELRPN:     24 atoms have been selected out of     24
 ( MAIN GRAP 1    A    )    1.0000    
 ( MAIN GRAP 1    B    )    1.0000    
 ( MAIN GRAP 1    C    )    1.0000    
 ( MAIN GRAP 1    D    )    1.0000    
 ( MAIN GRAP 1    E    )    1.0000    
 ( MAIN GRAP 1    F    )    2.0000    
 ( MAIN GRAP 1    G    )    2.0000    
 ( MAIN GRAP 1    H    )    2.0000    
 ( MAIN GRAP 1    I    )    3.0000    
 ( MAIN GRAP 1    J    )    3.0000    
 ( MAIN GRAP 1    K    )    3.0000    
 ( MAIN GRAP 1    L    )    4.0000    
 ( MAIN GRAP 2    A    )    5.0000    
 ( MAIN GRAP 2    B    )    5.0000    
 ( MAIN GRAP 2    C    )    5.0000    
 ( MAIN GRAP 2    D    )    5.0000    
 ( MAIN GRAP 2    E    )    5.0000    
 ( MAIN GRAP 2    F    )    6.0000    
 ( MAIN GRAP 2    G    )    6.0000    
 ( MAIN GRAP 2    H    )    6.0000    
 ( MAIN GRAP 2    I    )    7.0000    
 ( MAIN GRAP 2    J    )    7.0000    
 ( MAIN GRAP 2    K    )    7.0000    
 ( MAIN GRAP 2    L    )    8.0000    
 CNSsolve> 
 CNSsolve>connectivity 
 CONNECT>    print=true selection=(all) 
 SELRPN:     24 atoms have been selected out of     24
 CONNECT>    setidto=rmsd  nsetto=fbeta 
 CONNECT>end 
 CONNECt: selected atoms form      8 covalently disconnected set(s)

 CONNECt: list of all atoms in set      1
              (MAIN 1    GRAP A   )
              (MAIN 1    GRAP B   )
              (MAIN 1    GRAP C   )
              (MAIN 1    GRAP D   )
              (MAIN 1    GRAP E   )

 CONNECt: list of all atoms in set      2
              (MAIN 1    GRAP F   )
              (MAIN 1    GRAP G   )
              (MAIN 1    GRAP H   )

 CONNECt: list of all atoms in set      3
              (MAIN 1    GRAP I   )
              (MAIN 1    GRAP J   )
              (MAIN 1    GRAP K   )

 CONNECt: list of all atoms in set      4
              (MAIN 1    GRAP L   )

 CONNECt: list of all atoms in set      5
              (MAIN 2    GRAP A   )
              (MAIN 2    GRAP B   )
              (MAIN 2    GRAP C   )
              (MAIN 2    GRAP D   )
              (MAIN 2    GRAP E   )

 CONNECt: list of all atoms in set      6
              (MAIN 2    GRAP F   )
              (MAIN 2    GRAP G   )
              (MAIN 2    GRAP H   )

 CONNECt: list of all atoms in set      7
              (MAIN 2    GRAP I   )
              (MAIN 2    GRAP J   )
              (MAIN 2    GRAP K   )

 CONNECt: list of all atoms in set      8
              (MAIN 2    GRAP L   )
 CNSsolve> 
 CNSsolve>show elem ( rmsd ) ( all ) 
 SELRPN:     24 atoms have been selected out of     24
 ( MAIN GRAP 1    A    )    1.0000    
 ( MAIN GRAP 1    B    )    1.0000    
 ( MAIN GRAP 1    C    )    1.0000    
 ( MAIN GRAP 1    D    )    1.0000    
 ( MAIN GRAP 1    E    )    1.0000    
 ( MAIN GRAP 1    F    )    2.0000    
 ( MAIN GRAP 1    G    )    2.0000    
 ( MAIN GRAP 1    H    )    2.0000    
 ( MAIN GRAP 1    I    )    3.0000    
 ( MAIN GRAP 1    J    )    3.0000    
 ( MAIN GRAP 1    K    )    3.0000    
 ( MAIN GRAP 1    L    )    4.0000    
 ( MAIN GRAP 2    A    )    5.0000    
 ( MAIN GRAP 2    B    )    5.0000    
 ( MAIN GRAP 2    C    )    5.0000    
 ( MAIN GRAP 2    D    )    5.0000    
 ( MAIN GRAP 2    E    )    5.0000    
 ( MAIN GRAP 2    F    )    6.0000    
 ( MAIN GRAP 2    G    )    6.0000    
 ( MAIN GRAP 2    H    )    6.0000    
 ( MAIN GRAP 2    I    )    7.0000    
 ( MAIN GRAP 2    J    )    7.0000    
 ( MAIN GRAP 2    K    )    7.0000    
 ( MAIN GRAP 2    L    )    8.0000    
 CNSsolve>show elem ( fbeta ) ( all ) 
 SELRPN:     24 atoms have been selected out of     24
 ( MAIN GRAP 1    A    )    5.0000    
 ( MAIN GRAP 1    B    )    5.0000    
 ( MAIN GRAP 1    C    )    5.0000    
 ( MAIN GRAP 1    D    )    5.0000    
 ( MAIN GRAP 1    E    )    5.0000    
 ( MAIN GRAP 1    F    )    3.0000    
 ( MAIN GRAP 1    G    )    3.0000    
 ( MAIN GRAP 1    H    )    3.0000    
 ( MAIN GRAP 1    I    )    3.0000    
 ( MAIN GRAP 1    J    )    3.0000    
 ( MAIN GRAP 1    K    )    3.0000    
 ( MAIN GRAP 1    L    )    1.0000    
 ( MAIN GRAP 2    A    )    5.0000    
 ( MAIN GRAP 2    B    )    5.0000    
 ( MAIN GRAP 2    C    )    5.0000    
 ( MAIN GRAP 2    D    )    5.0000    
 ( MAIN GRAP 2    E    )    5.0000    
 ( MAIN GRAP 2    F    )    3.0000    
 ( MAIN GRAP 2    G    )    3.0000    
 ( MAIN GRAP 2    H    )    3.0000    
 ( MAIN GRAP 2    I    )    3.0000    
 ( MAIN GRAP 2    J    )    3.0000    
 ( MAIN GRAP 2    K    )    3.0000    
 ( MAIN GRAP 2    L    )    1.0000    
 CNSsolve> 
 CNSsolve>connectivity 
 CONNECT>    print=true selection=(all) 
 SELRPN:     24 atoms have been selected out of     24
 CONNECT>    setidto=store1  nsetto=store2 
 CONNECT>end 
 CONNECt: selected atoms form      8 covalently disconnected set(s)

 CONNECt: list of all atoms in set      1
              (MAIN 1    GRAP A   )
              (MAIN 1    GRAP B   )
              (MAIN 1    GRAP C   )
              (MAIN 1    GRAP D   )
              (MAIN 1    GRAP E   )

 CONNECt: list of all atoms in set      2
              (MAIN 1    GRAP F   )
              (MAIN 1    GRAP G   )
              (MAIN 1    GRAP H   )

 CONNECt: list of all atoms in set      3
              (MAIN 1    GRAP I   )
              (MAIN 1    GRAP J   )
              (MAIN 1    GRAP K   )

 CONNECt: list of all atoms in set      4
              (MAIN 1    GRAP L   )

 CONNECt: list of all atoms in set      5
              (MAIN 2    GRAP A   )
              (MAIN 2    GRAP B   )
              (MAIN 2    GRAP C   )
              (MAIN 2    GRAP D   )
              (MAIN 2    GRAP E   )

 CONNECt: list of all atoms in set      6
              (MAIN 2    GRAP F   )
              (MAIN 2    GRAP G   )
              (MAIN 2    GRAP H   )

 CONNECt: list of all atoms in set      7
              (MAIN 2    GRAP I   )
              (MAIN 2    GRAP J   )
              (MAIN 2    GRAP K   )

 CONNECt: list of all atoms in set      8
              (MAIN 2    GRAP L   )
 CNSsolve> 
 CNSsolve>show elem ( store1 ) ( all ) 
 SELRPN:     24 atoms have been selected out of     24
 ( MAIN GRAP 1    A    )    1.0000    
 ( MAIN GRAP 1    B    )    1.0000    
 ( MAIN GRAP 1    C    )    1.0000    
 ( MAIN GRAP 1    D    )    1.0000    
 ( MAIN GRAP 1    E    )    1.0000    
 ( MAIN GRAP 1    F    )    2.0000    
 ( MAIN GRAP 1    G    )    2.0000    
 ( MAIN GRAP 1    H    )    2.0000    
 ( MAIN GRAP 1    I    )    3.0000    
 ( MAIN GRAP 1    J    )    3.0000    
 ( MAIN GRAP 1    K    )    3.0000    
 ( MAIN GRAP 1    L    )    4.0000    
 ( MAIN GRAP 2    A    )    5.0000    
 ( MAIN GRAP 2    B    )    5.0000    
 ( MAIN GRAP 2    C    )    5.0000    
 ( MAIN GRAP 2    D    )    5.0000    
 ( MAIN GRAP 2    E    )    5.0000    
 ( MAIN GRAP 2    F    )    6.0000    
 ( MAIN GRAP 2    G    )    6.0000    
 ( MAIN GRAP 2    H    )    6.0000    
 ( MAIN GRAP 2    I    )    7.0000    
 ( MAIN GRAP 2    J    )    7.0000    
 ( MAIN GRAP 2    K    )    7.0000    
 ( MAIN GRAP 2    L    )    8.0000    
 CNSsolve>show elem ( store2 ) ( all ) 
 SELRPN:     24 atoms have been selected out of     24
 ( MAIN GRAP 1    A    )    5.0000    
 ( MAIN GRAP 1    B    )    5.0000    
 ( MAIN GRAP 1    C    )    5.0000    
 ( MAIN GRAP 1    D    )    5.0000    
 ( MAIN GRAP 1    E    )    5.0000    
 ( MAIN GRAP 1    F    )    3.0000    
 ( MAIN GRAP 1    G    )    3.0000    
 ( MAIN GRAP 1    H    )    3.0000    
 ( MAIN GRAP 1    I    )    3.0000    
 ( MAIN GRAP 1    J    )    3.0000    
 ( MAIN GRAP 1    K    )    3.0000    
 ( MAIN GRAP 1    L    )    1.0000    
 ( MAIN GRAP 2    A    )    5.0000    
 ( MAIN GRAP 2    B    )    5.0000    
 ( MAIN GRAP 2    C    )    5.0000    
 ( MAIN GRAP 2    D    )    5.0000    
 ( MAIN GRAP 2    E    )    5.0000    
 ( MAIN GRAP 2    F    )    3.0000    
 ( MAIN GRAP 2    G    )    3.0000    
 ( MAIN GRAP 2    H    )    3.0000    
 ( MAIN GRAP 2    I    )    3.0000    
 ( MAIN GRAP 2    J    )    3.0000    
 ( MAIN GRAP 2    K    )    3.0000    
 ( MAIN GRAP 2    L    )    1.0000    
 CNSsolve> 
 CNSsolve> 
 CNSsolve>! 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8201488 bytes
           Maximum dynamic memory overhead:           320 bytes
           Program started at: 16:07:50 on 05-Jul-2018
           Program stopped at: 16:07:50 on 05-Jul-2018
           CPU time used:       0.0072 seconds
          ============================================================
